src/web/core/src/main/java/org/geoserver/web/wicket/FileExistsValidator.java

```diff
@@ -88,7 +89,8 @@ public void validate(IValidatable<String> validatable) {
         GeoServerResourceLoader loader = GeoServerExtensions.bean(GeoServerResourceLoader.class);
         if (baseDirectory != null) {
             // local to provided baseDirectory
-            relFile = Files.url(baseDirectory, uriSpec);
+            relFile = Resources.find(Resources.fromURL(Files.asResource(baseDirectory), uriSpec),
+                    true);
```

Comment:
Technically this call with force the file to exist on the file system (is it available in the configuration system).

Reply:
Try to untangle the previous code, it should do the same (this is the result of a few automatic inline refactors). force=true makes it return a File object even if the file is not on disk, without materializing one. If the resource is there instead, the previous code would have materialized it on file system already. No changes here, if it's broken now it was likely broken before too.

Reply:
Oh gak, I assumed force=true would create a file if needed.